datasetinsights/stats/visualization/plots.py

```diff
@@ -370,3 +372,90 @@ def rotation_plot(df, x, y, z=None, max_samples=None, title=None, **kwargs):
         )
     )
     return fig
+
+
+def model_performance_comparison_box_plot(
+    title,
+    mean_ap_base=None,
+    mean_ap_50_base=None,
+    mean_ar_base=None,
+    mean_ap_new=None,
+    mean_ap_50_new=None,
+    mean_ar_new=None,
+    range=[0, 1.0],
+    **kwargs,
+):
+    """Create a box plot for a base and new model performance
+    Args:
+        title (str): title of the plot
+        mean_ap_base (list): a list of base mAP, [0.1, 0.2, ...]
+        mean_ap_50_base (list): a list of base mAP, [0.4, 0.5, ...]
```

Comment:
Do we need this  [0.1, 0.2, ...]? list of mAP is self explanatory. or if you want to include ex then avoid using "..."